### PR TITLE
How to Build an Image Classifier with Keras

### PR DESCRIPTION
**Pull request by Inyrkz**
update branch

#### Preliminary Checks - Formatting and Structure 
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.

**NOTE: (1st time contributors - submit ONLY one article to get familiar with the process and style-once published, you can then submit any number of articles.)**
      
#### Grammar Checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical Checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution Guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md)**

### CODE REVIEW

**Comment by lalith1403** [comment]
Change the title of the pr to title of the article. Also, check the boxes in the checklist whichever is applicable.

**Comment by Inyrkz** [comment]
Added a folder image-classifier-keras.
Wrote an article about how to build an Image Classifier using Keras with a TensorFlow backend.
There are images of code output in the folder along with memes to aid explanation. 
Some images are original while others are from the internet, the sources have been provided.

**Comment by lalith1403** [comment]
Can you add a hero image?

**Comment by Inyrkz** [comment]
This fixes #1072

**Comment by hectorkambow** [comment]
plagiarism check done

File: articles/image-classifier-keras/index.md [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+- Prediction on New Data
+
+ 
+### Prerequisites
+- Basics of Convolutional Neural Network. I recommend this [article](https://www.section.io/engineering-education/basics-of-convolution-neural-networks/) by Willies Ogola to get started.
+- Python Programming
+- [Colab Notebook](https://colab.research.google.com/)
+
+
+### Importing libraries and exploring the dataset
```
Comment (lalith1403) [inline articles/image-classifier-keras/index.md:25]: Before you begin with the tutorial, ensure you insert a link to your Colab file with the outputs. This way, a person who wants to check your project can do so without having to go through the entire article,

File: articles/image-classifier-keras/index.md [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+
+### Build CNN model
+
+```python
+# Building CNN Model
+model = Sequential()
+model.add(Conv2D(32, (5,5), padding='same', activation='relu',
+                input_shape=(200, 200, 3)))
+model.add(MaxPooling2D(pool_size=(2,2)))
+model.add(Dropout(0.5))
```
Comment (lalith1403) [inline articles/image-classifier-keras/index.md:299]: My experience says a dropout of 0.2-0.25 is ideal. Anything above that leads to loss of data and bad training. 0.5 is huge,

**Review by lalith1403 — CHANGES_REQUESTED** [review]
Well written article. Implement the changes and we can move onto the final round of review.

**Comment by lalith1403** [comment]
Also, we will need a short bio and a headshot pic of you. Look at the authors folder for more information.

File: articles/image-classifier-keras/index.md [hunk 3]
```diff
@@ -0,0 +1,488 @@
...
+                                               target_size=TARGET_SIZE,
+                                               batch_size=BATCH_SIZE,
+                                               class_mode=CLASS_MODE)
+```
+
+![alt text](/engineering-education/image-classifier-keras/11.jpg)
+
+The first argument is the path to the dataset. The next parameter is the *target_size*. It will resize all the images to the specified target size of 200x200. The *batch size* defines how many images we want to have in each batch. 
+
+We will use a batch size of 32, and the class mode is either *binary* or *categorical*. *Binary* is for two output classes, while categorical is for more than two classes. Since we are working with only *two* classes, we will set the class mode to *binary*.
```
Comment (lalith1403) [inline articles/image-classifier-keras/index.md:270]: Don't use future tense. Instead, write it as: We use a batch size of 32... I observed it throughout your article. Needs to be changed at all occurrences.

**Comment by lalith1403** [comment]
Are the changes done? Notify us after all the changes are made.

**Comment by Inyrkz** [comment]
All the changes have been made.

**Comment by Inyrkz** [comment]
All the changes have been made.
